src/neha_Patil/BankSystem.java

```diff
@@ -0,0 +1,94 @@
+package neha_Patil;
+
+public class BankSystem {
+	int CreditAmt;
+	int DebitAmt;
+	String Cname;
+	int Totalbalance = 10000;
+ 
+
+	int bal;
+	int CreditCount;
+	int DebitCount;
+
+	static int totaldebitcount = 0;
+	static int totalcreditcount = 0;
+	static int totalbalcount = 0;
+
+	
+
+	void Credit(String CustName, int Camt) {
+		Cname = CustName;
+		CreditAmt = Camt;
+		System.out.println("Credit Amount  :" + Camt);
+		Totalbalance = Totalbalance + CreditAmt;
+		CreditCount++;
+		totalcreditcount++;
+	}
+
+
+	void Debit(String CustName, int Damt) {
+ 
+
+		Cname = CustName;
```

Comment:
You should handle below condition,

if(requestAmt <= totalAmt) then only you can deduct amt else not